splinterd/Cargo.toml

```diff
@@ -41,7 +41,7 @@ cylinder = "0.2"
 flexi_logger = "0.14"
 health = { path = "../services/health", optional = true }
 log = "0.4"
-openssl = { version = "0.10", optional = true }
```

Comment:
This might be better suited to be additionally added as a dev dependency or, there should be a feature that depends on this used by the tests.

Reply:
Updated to add `open-ssl` as a dev-dependency